spec/unit/views/components/sidebar_section_spec.rb

```diff
@@ -11,36 +11,37 @@
   end
 
   let(:html) do
-    render_arbre_component section: section do
+    result = render_arbre_component section: section do
       sidebar_section(assigns[:section])
     end
+    Capybara.string(result.to_s)
   end
 
   it "should have a title h3" do
-    expect(html.find_by_tag("h3").first.content).to eq "Help Section"
+    expect(html).to have_css '.panel-title', text: 'Help Section'
   end
 
-  it "should have the class of 'sidebar_section'" do
-    expect(html.class_list).to include("sidebar_section")
+  it "should have the class of 'sidebar-section'" do
+    expect(html).to have_css ".sidebar-section"
   end
 
   it "should have an id based on the title" do
-    expect(html.id).to eq "help-section_sidebar_section"
+    expect(html).to have_css "#help-section-sidebar-section"
   end
 
-  it "should have a contents div" do
-    expect(html.find_by_tag("div").first.class_list).to include("panel_contents")
+  it "should have a panel body" do
+    expect(html).to have_css '.panel-body'
   end
 
-  it "should add children to the contents div" do
-    expect(html.find_by_tag("span").first.parent).to eq html.find_by_tag("div").first
+  it "should add children to the panel body" do
+    expect(html).to have_css '.panel-body > span'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.